eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -212,6 +214,18 @@ conwayModifyPParams f = modifyNES $ \nes ->
         (snapshot, ratifyState) ->
           DRComplete snapshot (ratifyState & rsEnactStateL . ensCurPParamsL %~ f)
 
+initialCommitteeKeyPair :: Crypto c => KeyPair 'ColdCommitteeRole c
+initialCommitteeKeyPair = mkKeyPair 30
```

Comment:
These can be generated randomly, no need to use this hack of relying on an integer.

See my comment below on how to do that